src/objective/multiclass_obj.cu

```diff
@@ -101,7 +101,8 @@ class SoftmaxMultiClassObj : public ObjFunction {
           for (int k = 0; k < nclass; ++k) {
             // Computation duplicated to avoid creating a cache.
             bst_float p = expf(point[k] - wmax) / static_cast<float>(wsum);
-            const bst_float h = fmax(2.0f * p * (1.0f - p) * wt, kRtEps);
+            const float eps = 1e-16f;
```

Comment:
There is a eps defined in:
https://github.com/dmlc/xgboost/blob/516457fadcd315554e052b24307392b6aad30e49/include/xgboost/base.h#L203

Not very important. :)

Reply:
This is actually the Windows compilation error. For some reason this constant is not defined in device code. Also this number was supposed to be 1e-16f, kRtEps is for something different.

Reply:
@RAMitchell I'd like to get Windows slaves up and running soon, so that we can automatically check compilation issues.